cmd/ocp-build-data-enforcer/main.go

```diff
@@ -0,0 +1,252 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/openshift/imagebuilder"
+	dockercmd "github.com/openshift/imagebuilder/dockerfile/command"
+	"github.com/pmezard/go-difflib/difflib"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/errgroup"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"sigs.k8s.io/yaml"
```

Comment:
in general, in `ci-tools` repo we are using the `"github.com/ghodss/yaml"` library. If there is no good reason, I don't think we should import this one too.

Reply:
Doesn't look like that:
```
$ rg sigs.k8s.io/yaml pkg/ cmd/
pkg/load/load.go
16:	"sigs.k8s.io/yaml"

cmd/determinize-prow-config/main.go
16:	"sigs.k8s.io/yaml"

cmd/testgrid-config-generator/main.go
21:	"sigs.k8s.io/yaml"

pkg/dispatcher/config.go
10:	"sigs.k8s.io/yaml"

cmd/repo-init/main.go
24:	"sigs.k8s.io/yaml"

pkg/testhelper/testhelper.go
11:	"sigs.k8s.io/yaml"

cmd/determinize-peribolos/main.go
13:	"sigs.k8s.io/yaml"

cmd/sanitize-prow-jobs/main.go
15:	"sigs.k8s.io/yaml"

cmd/bugzilla-backporter/main.go
24:	"sigs.k8s.io/yaml"

cmd/prow-job-dispatcher/main.go
23:	"sigs.k8s.io/yaml"

cmd/registry-replacer/main.go
26:	"sigs.k8s.io/yaml"

cmd/ocp-build-data-enforcer/types_test.go
6:	"sigs.k8s.io/yaml"

cmd/ocp-build-data-enforcer/main.go
20:	"sigs.k8s.io/yaml"

pkg/util/imagestreamtagwrapper/imagestreamtagwrapper_test.go
13:	"sigs.k8s.io/yaml"

pkg/controller/promotionreconciler/reconciler_test.go
19:	"sigs.k8s.io/yaml"
```

`sigs.k8s.io/yaml` is what is used everywhere in the ecosystem so I think it makes more sense to stick with it. Also, it is a transient dependency anyways. If anything, we should remove `github.com/ghodss/yaml` usage